restapi/api/routes.go

```diff
@@ -17,22 +18,19 @@ func registerRoutes(router *gin.RouterGroup) {
 	device.PUT("/enable-condition", EnableDeviceCondition)
 	device.POST("/disable-condition", DisableDeviceCondition)
 
-	router.GET("/reserved-devices", reservation.GetReservedDevices)
-	reservations := router.Group("/reserve/:udid")
-	reservations.Use(DeviceMiddleware())
-	reservations.POST("/", reservation.ReserveDevice)
-	reservations.DELETE("/", reservation.ReleaseDevice)
+	device.POST("/reserve", ReserveDevice)
+	device.DELETE("/reserve", ReleaseDevice)
```

Comment:
I just realized this now, the DELETE endpoint needs to be connected to the UUID token you get for a reservation. Otherwise anyone could just delete my device reservations. 
so it should be DELETE /reservations/:token.
Technically you do not need the device UDID for it.

Reply:
yep, that was on my mind when adding the endpoints but completely forgot about it, good point!

Reply:
I think it makes sense to keep it so we can properly report if the device is no longer connected through the `DeviceMiddleware` and it's also easier to check the map with reserved devices having it. Still let me know if you prefer not having it and I'll rework it

Reply:
Now that I think about it, we shouldn't care if its connected or not, we just want to clear it from the reserved devices list. I'll update it

Reply:
done